Jenkinsfile

```diff
@@ -17,13 +20,18 @@ pipeline {
             }
         }
     }
-
+	  
+	
     stage('Create memphis namespace in Kubernetes'){
         steps {
             sh """
+            sudo yum install lsof -y
```

Comment:
Please add it to image @bkochauri-memphis

Reply:
@valeraBr  Its done.